community/kernel/src/docs/dev/transactions.asciidoc

```diff
@@ -61,6 +61,36 @@ In addition, the Neo4j Java API (see <<advanced-usage>>) enables explicit lockin
 Using locks gives the opportunity to simulate the effects of higher levels of isolation by obtaining and releasing locks explicitly.
 For example, if a write lock is taken on a common node or relationship, then all transactions will serialize on that lock -- giving the effect of a serialization isolation level.
 
+=== Lost Updates in Cypher ===
+
+In cypher it is possible to acquire write locks to simulate improved isolation in some cases.
+Consider, for example, the case where multiple concurrent cypher queries increment the value of a property.
+Due to the read-committed isolation level, the increments will not results in a deterministic final solution.
+
+For example, the following query, if run by one hundred concurrent clients, will not increment the property `n.prop` to 100, but some value lower than 100.
```

Comment:
Though very unlikely, but the final result might actually be `100` for some runs. I would suggest to rephrase accordingly.